packages/mgt-components/package.json

```diff
@@ -40,7 +40,7 @@
     "@microsoft/microsoft-graph-client": "^2.2.1",
     "@microsoft/microsoft-graph-types": "^2.0.0",
     "@microsoft/microsoft-graph-types-beta": "^0.16.0-preview",
-    "@fluentui/web-components": "0.22.1",
+    "@fluentui/web-components": "2.5.3",
```

Comment:
This is a big jump in version here, have we validated that there are no-breaking changes introduced for our library consumers?

Reply:
Moving this PR to point to next/fluentui branch.